<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Golem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewCollection"/>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Database"/>
    <addaction name="actionImport_Database"/>
    <addaction name="separator"/>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEdit_Collection"/>
    <addaction name="actionRemove_Collection"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Project"/>
    <addaction name="actionRemove_Project"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewCollection"/>
   <addaction name="actionEdit_Collection"/>
   <addaction name="actionRemove_Collection"/>
   <addaction name="separator"/>
   <addaction name="actionNewProject"/>
   <addaction name="actionEdit_Project"/>
   <addaction name="actionRemove_Project"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Events</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEditFilterEvents">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidgetEvents">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>All Events</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEditCollections">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidgetCollections">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>My Collections</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/folder--plus.png</normaloff>:/png/icons/folder--plus.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="iconText">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewCollection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/box--plus.png</normaloff>:/png/icons/box--plus.png</iconset>
   </property>
   <property name="text">
    <string>New Collection</string>
   </property>
   <property name="iconText">
    <string>New Collection</string>
   </property>
   <property name="toolTip">
    <string>New Collection</string>
   </property>
  </action>
  <action name="actionEdit_Collection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/box--pencil.png</normaloff>:/png/icons/box--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit Collection</string>
   </property>
  </action>
  <action name="actionRemove_Collection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/box--minus.png</normaloff>:/png/icons/box--minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Collection</string>
   </property>
  </action>
  <action name="actionEdit_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/folder--pencil.png</normaloff>:/png/icons/folder--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit Project</string>
   </property>
  </action>
  <action name="actionRemove_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/folder--minus.png</normaloff>:/png/icons/folder--minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Project</string>
   </property>
  </action>
  <action name="actionExport_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/database--arrow.png</normaloff>:/png/icons/database--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Export Database</string>
   </property>
  </action>
  <action name="actionImport_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/png/icons/database-import.png</normaloff>:/png/icons/database-import.png</iconset>
   </property>
   <property name="text">
    <string>Import Database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Golem</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
